tests/tutor/assignment/test_assignment_reading.py

```diff
@@ -40,8 +40,8 @@ def test_edit_published_reading(tutor_base_url, selenium, teacher):
     """Test editing the published reading."""
     # GIVEN: Logged into Tutor as a teacher
     # AND: Has an existing course
-    # AND: Has a publish reading
-
+    # AND: Has a published reading
+    
```

Comment:
Watch out for random bits of whitespace; most editors have a setting to clear the excess space when the line in question loses focus and/or when you save the file.